python/artm/score_tracker.py

```diff
@@ -128,16 +135,21 @@ def __init__(self, score):
         :Properties:
         * Note: every field is a list of info about score on all synchronizations.
         * value - values of perplexity.
-        * raw - raw values in formula for perplexity.
-        * normalizer - normalizer values in formula for perplexity.
-        * zero_tokens - number of zero p(w|d) = sum_t p(w|t) p(t|d).
+        * raw - raw values in formula for perplexity (in case of one class id).
+        * normalizer - normalizer values in formula for perplexity  (in case of one class id).
+        * zero_tokens - number of zero p(w|d) = sum_t p(w|t) p(t|d)  (in case of one class id).
+        * class_id_info - array of structures, each structure contains raw, normalizer\
+                          zero_tokens and class_id name (in case of several class ids).
```

Comment:
What about excluding `class_id` from `ClassInfo` containter and add `repeated class_id` into `PerplexityScore`? In that case we can reuse the resulting container to show score for all class-ids.